qutip/solver/flimesolve.py

```diff
@@ -59,107 +59,103 @@ def _floquet_rate_matrix(floquet_basis,
     '''
     Hdim = len(floquet_basis.e_quasi)
 
-    time = T
+    time = floquet_basis.T
     dt = time / Nt
     tlist = np.linspace(0, time - dt, Nt)
+    omega = 2 * np.pi / time
```

Comment:
`omega` could not be the first entry in args. Are there any case where it could be independent from the period?

Reply:
I've made it so that omega is directly pulled from the period going forward.